packages/kbn-ai-playground/components/summarization_panel/open_ai_summarization_model.tsx

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useState } from "react";
+
+import {
+  EuiButtonEmpty,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiFormRow,
+  EuiIcon,
+  EuiSuperSelect,
+  EuiSuperSelectOption,
+  EuiToolTip
+} from "@elastic/eui";
+
+import { i18n } from "@kbn/i18n";
+
+import { OpenAIIcon } from "./open_ai_icon";
+
+const renderSelectOptions = (label: string) => (
+  <EuiFlexGroup alignItems="center">
+    <EuiFlexItem grow={false}>
+      <EuiIcon type={OpenAIIcon} />
+    </EuiFlexItem>
+    <EuiFlexItem>
+      {label}
+    </EuiFlexItem>
+  </EuiFlexGroup>
+);
+
+
+enum SummarizationModelName {
+  gpt3_5 = 'gpt-3.5-turbo',
+  gpt3_5_turbo_1106 = 'gpt-3.5-turbo-1106',
+  gpt3_5_turbo_16k = 'gpt-3.5-turbo-16k',
+  gpt3_5_turbo_16k_0613 = 'gpt-3.5-turbo-16k-0613',
+  gpt3_5_turbo = 'gpt-3.5-turbo-instruct',
+}
+
+const Summarization_Model: EuiSuperSelectOption<string>[] = [
```

Comment:
And small things the naming of const and values suppose to follow CamelCase or camelCase format